kbase-extension/static/kbase/js/widgets/appWidgets2/input/checkboxInput.js

```diff
@@ -86,44 +90,106 @@ define([
 
         // RENDERING
 
+        function clearInitialValueError() {
+            const inputContainer = container.querySelector(`.${cssBaseClass}_container`);
+            inputContainer.classList.remove(cssErrorClass);
+            inputContainer.querySelector(`.${cssBaseClass}_error`).remove();
+            model.hasInitialValueError = false;
+        }
+
+        function renderInitialValueError(events) {
+            const defaultVal =
```

Comment:
The actual error stuff. Show some text similar to other input widgets, with a button on the right side.